pkg/ccl/streamingccl/streamingest/stream_ingestion_planning.go

```diff
@@ -203,11 +203,13 @@ func ingestionPlanHook(
 		}
 
 		jr := jobs.Record{
-			Description:   jobDescription,
-			Username:      p.User(),
-			Progress:      jobspb.StreamIngestionProgress{},
-			Details:       streamIngestionDetails,
-			NonCancelable: true,
+			Description: jobDescription,
+			Username:    p.User(),
+			Progress:    jobspb.StreamIngestionProgress{},
+			Details:     streamIngestionDetails,
+			// TODO(casper): make it retryable with exponential backoff and
+			// finally pause on error.
+			NonCancelable: false,
```

Comment:
This is a fine change. But you might want to mention it in the commit message.

Reply:
It's also default; perhaps drop the noncancellable bit?